dist/access_check/check_sources.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import sys, os
+from layercparse import *
+from pprint import pprint, pformat
+
+def main():
+    # setLogLevel(LogLevel.WARNING)
```

Comment:
Should this become an option to set the log level so you can uncomment the other lines further down?